frontend/src/features/manageProfile/components/dashboard/ManageProfilesDashboard.tsx

```diff
@@ -97,7 +97,7 @@ export const ManageProfilesDashboard = React.memo(() => {
       : null,
     shouldAllowUserManagement
       ? {
-          label: "User Management",
+          label: "Add / Manage Users",
```

Comment:
Needs a change in the breadcrumb too?
https://github.com/bcgov/onroutebc/blob/main/frontend/src/features/manageProfile/pages/EditUserDashboard.tsx#L93